website/screens/components/sidenav/usage/SidenavUsagePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { DxcText, DxcList, DxcStack, DxcTable } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import HeadingLink from "../../../common/HeadingLink";
+import sidenavVariants from "./images/sidenav_variants.png";
+import Figure from "../../../common/Figure";
+import DocFooter from "../../../common/DocFooter";
+
+const SidenavUsagePage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Usage</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Use the sidenav element to improve the discoverability of the application, making the
+            navigation links accessible to the users.
+          </DxcText>
+          <DxcText>
+            Keep in mind the type of the devices that you are developing for, and handle the
+            behavior in a way that doesn&#39;t block or reduce the available space of the main area
```

Comment:
Remove `&#39;` and use apostrophe.

Reply:
This is the same as an apostrophe and avoids triggering an error in visual studio.

Reply:
It doesn't trigger any error in VS.
![image](https://user-images.githubusercontent.com/35959222/157406635-95d8e053-2e98-41d0-89d1-a82a3cd99b97.png)

Reply:
I don't think this should be changed since it works the same way and also avoids errors in some VS.
Also it seems that there are multiple cases in which the code is used over the apostrophe.